packages/mui-material/src/Dialog/DialogContext.ts

```diff
@@ -1,10 +1,10 @@
-import { createContext } from 'react';
+import * as React from 'react';
```

Comment:
This case is interesting. I think that the initial problem was that:

```jsx
import ReactDOMServer from 'react-dom/server';
```

is wrong because React plans to no longer export default. This change goes a step further into normalization, why not.